package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Tue Jan 21 09:43:35 UTC 2025 - Ancor Gonzalez Sosa <ancor@suse.com>
+
+- Pervasive encryption: support for AES-CIPHER and EP11 and better
```

Comment:
I would probably use `improve` instead of `better`

Reply:
I used "better" because it's shorter and fits nicely in the line. ;-) But I can change it.